tools/rna_tools/sortmerna/sortmerna.xml

```diff
@@ -246,8 +346,9 @@ The user may also choose to use their own rRNA databases.
 
 .. class:: warningmark
 
-Note that your personal databases are indexed each time, and that
+Note that the databases are indexed each time given the choosed indexation parameters, and that
```

Comment:
s/indexation/indexing/

Also this should be updated to reflect the latest changes.